src/UI/ComposeEditor/Main.hs

```diff
@@ -51,7 +51,9 @@ attachmentsEditor s =
     in attachmentsHeader <=> attachmentsList
 
 drawHeaders :: AppState -> Widget Name
-drawHeaders s = padBottom (Pad 1) $ foldr (drawTableRows s) (txt T.empty) [ComposeSubject, ComposeTo, ComposeFrom]
+drawHeaders s = 
+  let headers' = [ComposeSubject, ComposeTo, ComposeFrom, ComposeCc, ComposeBcc]
```

Comment:
At the moment the fields are rendered like so:
![screenshot](https://user-images.githubusercontent.com/121047/74206345-bc9f1800-4cc6-11ea-82b7-2aea75dfb388.png)

I suspect if you stick the `CC` and `BCC` `Name`s between `ComposeSubject` and `ComposeTo` they will be rendered below the `To:` line. I think that would be the better place since they're not as often used as `To` and `From`